.travis.yml

```diff
@@ -191,23 +185,43 @@ matrix:
           compiler: clang
 
         # Also regularly try the big-endian s390 architecture, in the
-        # process checking that installing dependencies with apt works.
+        # process checking that installing dependencies with apt works,
+        # and that astropy can properly use the system libraries.
+        # The latter is easy since the python3-astropy package, which is
+        # used to install the regular dependencies, also pulls those in.
+        # Hence, one can and should use ASTROPY_USE_SYSTEM_ALL=1.
         - name: big-endian s390x architecture with apt
           arch: s390x
           language: c
-          dist: bionic
-          stage: Cron tests
+          dist: focal
+          stage: Initial tests
           env: SETUP_METHOD='apt'
                ASTROPY_USE_SYSTEM_ALL=1
+          addons:
+              apt:
+                  packages:
+                      - python3-astropy
+                      - python3-venv
+                      - cython3
+                      - wcslib-dev
+                      - libcfitsio-dev
```

Comment:
Why install astropy here ? Also cython should be installed with the other build requirements by pip.

Reply:
I should add a comment: the nice thing of doing that is that it automatically pulls in `numpy` and `erfa` (`wcslib` and `libcfitsio` as well, but those also get pulled in by their `-dev`) and everything else astropy needs. And with that, I can use the same tests to check that we can use system libraries (both C libraries, and via `--use-system-libraries`, python packages). I.e., I try here to let `pip` do as little as possible, with the hope that these tests will make packaging on linux systems easier (and, selfishly, my own setup!).

Reply:
explicit is still better than implicit, I vaguely recall it bite back to rely on this automated pull of packages rather than spelling them out (I have nothing against installing numpy/erfa/etc from the linux system packaging rather than pip)

Reply:
@bsipocz - fair point! Happy to change. (Though what I have here also has its advantages in being realistic - at least, taking myself as an example, I do have astropy installed from the distribution while working on development...)

Reply:
OK, added explicit dependencies (now testing whether I didn't mess up...)